python_modules/dagster/dagster/_core/execution/plan/execute_plan.py

```diff
@@ -42,8 +42,8 @@ def inner_plan_execution_iterator(
 
         # begin capturing logs for the whole process if this is a captured log manager
         if isinstance(compute_log_manager, CapturedLogManager):
-            pid = str(os.getpid())
-            log_key = compute_log_manager.build_log_key_for_run(pipeline_context.run_id, pid)
+            file_key = create_compute_log_file_key()
+            log_key = compute_log_manager.build_log_key_for_run(pipeline_context.run_id, file_key)
```

Comment:
should we include step key at all when we have a single step key? i guess no real reason to bother its not that useful